clients/product-settings/src/SettingsClient.ts

```diff
@@ -151,7 +151,7 @@ export class ConnectSettingsClient extends Client implements SettingsAdmin {
     requestContext.enter();
     const baseUrl: string = await this.getUrl(requestContext);
     requestContext.enter();
-    const accessTokenString: string | undefined = requestContext.accessToken.toTokenString();
+    const accessTokenString: string | undefined = requestContext.accessToken;

```

Comment:
why not type this as AccessTokenString ?

Reply:
same comment for everywhere else this is applicable

Reply:
Guess I didn't catch that when editing all the access token references. Should be updated now.